subprojects/docs/src/main/resources/header.html

```diff
@@ -165,50 +164,98 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
+                    <li><a href="viewing_debugging_dependencies.html">Debugging Dependencies</a></li>
+                    <li><a href="dependency_resolution.html">Dependency Resolution</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#advanced-techniques" aria-expanded="false" aria-controls="advanced-techniques">Advanced Techniques</a>
-                <ul id="advanced-techniques">
-                    <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing Parallel Tasks</a></li>
-                    <li><a href="test_kit.html">Testing Your Build with TestKit</a></li>
-                    <li><a href="ant.html">Using Ant from Gradle</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#declaring-dependency-versions" aria-expanded="false" aria-controls="declaring-dependency-versions">Choosing Versions</a>
+                <ul id="declaring-dependency-versions">
+                    <li><a href="single_versions.html">Declaring a Simple Version</a></li>
+                    <li><a href="rich_versions.html">Declaring a Rich Version</a></li>
+                    <li><a href="dynamic_versions.html">Handling Changing Versions</a></li>
+                    <li><a href="dependency_locking.html">Locking Versions</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
-                <ul id="sample-gradle-builds">
-                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
-                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#controlling-transitive-dependencies" aria-expanded="false" aria-controls="controlling-transitive-dependencies">Controlling Transitives</a>
+                <ul id="controlling-transitive-dependencies">
+                    <li><a href="dependency_constraints.html">Declaring Constraints</a></li>
+                    <li><a href="platforms.html">Using Platforms</a></li>
+                    <li><a href="dependency_alignment.html">Aligning Dependencies</a></li>
+                    <li><a href="dependency_capability_conflict.html">Managing Conflicting Dependencies</a></li>
+                    <li><a href="component_metadata_rules.html">Fixing Metadata</a></li>
+                    <li><a href="resolution_rules.html">Customizing Resolution</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#modeling-features" aria-expanded="false" aria-controls="modeling-features">Utilizing Variants</a>
+                <ul id="modeling-features">
```

Comment:
All the titles in this section use _solution_ wording instead of _problem_ wording. We should find better names.

Reply:
Titles changed as discussed